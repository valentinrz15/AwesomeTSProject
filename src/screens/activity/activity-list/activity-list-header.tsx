import React, { FunctionComponent } from 'react';
import { Text } from 'react-native';
import activityListStyles from './activity-list-styles';

interface ActivityListHeaderProps {
  title: string;
}

export const ActivityListHeader: FunctionComponent<ActivityListHeaderProps> = ({
  title,
}) => <Text style={activityListStyles.headerList}>{title}</Text>;
