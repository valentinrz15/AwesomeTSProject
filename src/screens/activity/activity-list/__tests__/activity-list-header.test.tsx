import React from 'react';
import renderer from 'react-test-renderer';
import { ActivityListHeader } from '../activity-list-header';

describe('activity list header test', () => {
  it('should render activity list header snapshot', () => {
    const { toJSON, root } = renderer.create(
      <ActivityListHeader title="Close" />,
    );

    expect(root.props.title).toBe('Close');

    expect(toJSON()).toMatchSnapshot();
  });
});
