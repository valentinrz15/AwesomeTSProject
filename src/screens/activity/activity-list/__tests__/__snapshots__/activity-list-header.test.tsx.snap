// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`activity list header test should render activity list header snapshot 1`] = `
<Text
  style={
    Object {
      "color": "#000",
      "fontSize": 22,
    }
  }
>
  Close
</Text>
`;
